tsconfig.json

```diff
@@ -9,12 +9,14 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "allowJs": true,
```

Comment:
can we remove this please, we don't want to be hacking our tsconfig to get it working with normal js code as well